front/src/Connexion/RoomConnection.ts

```diff
@@ -475,6 +481,13 @@ export class RoomConnection implements RoomConnection {
                     console.error("An error occurred server side: " + message.errorMessage.message);
                     break;
                 }
+                case "errorV2Message": {
+                    this._errorV2MessageStream.next(message.errorV2Message);
+                    if (message.errorV2Message.code !== "retry") this.closed = true;
+                    console.error("An error occurred server side: " + message.errorV2Message.code);
+                    errorScreenStore.setError(message.errorV2Message as unknown as WAError);
```

Comment:
```suggestion
                    errorScreenStore.setError(message.errorV2Message as unknown as WAError);
```

Do we really need this cast: ` as unknown as WAError` ?
Why?

Reply:
Wait.... actually, you should not be allowed to cast an interface to a Javascript class (`WAError`). Is this actually working?